tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests/AspireSqlServerEFCoreSqlClientExtensionsTests.cs

```diff
@@ -79,7 +79,7 @@ public void CanConfigureDbContextOptions()
         builder.Configuration.AddInMemoryCollection([
             new KeyValuePair<string, string?>("ConnectionStrings:sqlconnection", ConnectionString),
             new KeyValuePair<string, string?>("Aspire:Microsoft:EntityFrameworkCore:SqlServer:Retry", "true"),
-            new KeyValuePair<string, string?>("Aspire:Microsoft:EntityFrameworkCore:SqlServer:Timeout", "608")
+            new KeyValuePair<string, string?>("Aspire:Microsoft:EntityFrameworkCore:SqlServer:CommandTimeout", "608")
```

Comment:
The existing equivalent tests for oracle follows the same pattern:
https://github.com/radical/aspire/blob/3ffb92758f2c39b64fe56adc1ebd5269bd5c0473/tests/Aspire.Oracle.EntityFrameworkCore.Tests/AspireOracleEFCoreDatabaseExtensionsTests.cs#L76-L85

- timeout setting in in-memory collection for `CanConfigureDbContextOptions`
- and set via code in `CanConfigureDbContextOptionsWithoutRetry`

If not this pattern, then should I add a duplicate test for the new properties, and which one should it be a dupe of?

Reply:
.. and https://github.com/radical/aspire/blob/3ffb92758f2c39b64fe56adc1ebd5269bd5c0473/tests/Aspire.Oracle.EntityFrameworkCore.Tests/AspireOracleEFCoreDatabaseExtensionsTests.cs#L121-L137

Reply:
What is it you are trying to test? Just add a new test for that.

In general, when adding a new feature and updating existing tests, it is important to ensure you aren't losing coverage of existing scenarios.